ggshield/verticals/iac/filter.py

```diff
@@ -37,7 +37,8 @@ def get_iac_files_from_path(
         exclusion_regexes=exclusion_regexes,
         recursive=True,
         yes=True,
-        verbose=verbose,
+        display_binary_files=verbose,
+        display_scanned_files=False,  # this displays all file in the directory but we only want to display IaC files
```

Comment:
```suggestion
        display_scanned_files=False,   # If True, this would display all files in the directory but we only want to display IaC files 
```